script.js

```diff
@@ -0,0 +1,618 @@
+            createdAt: new Date('2021-03-12T23:00:00'),
+            link: 'link',
+            vendor: 'vendor',
+            hashTags: ['hashtag'],
+            discount: '10%',
+            validUntil: new Date('2021-03-29T23:00:00'),
+        },
+        
+        {
+            id: '14',
+            description: 'desctiption',
+            createdAt: new Date('2021-03-15T23:00:00'),
+            link: 'link',
+            vendor: 'vendor',
+            hashTags: ['hashtag'],
+            discount: '10%',
+            validUntil: new Date('2021-08-08T23:00:00'),
+        },
+        
+        {
+            id: '15',
+            description: 'desctiption',
+            createdAt: new Date('2021-03-19T15:00:00'),
+            link: 'link',
+            vendor: 'vendor',
+            hashTags: ['hashtag'],
+            discount: '10%',
+            validUntil: new Date('2021-07-01T23:00:00'),
+        },
+        
+        {
+            id: '16',
+            description: 'desctiption',
+            createdAt: new Date('2021-03-10T23:35:00'),
+            link: 'link',
+            vendor: 'vendor',
+            hashTags: ['hashtag'],
+            discount: '10%',
+            validUntil: new Date('2021-05-22T23:00:00'),
+        },
+        
+        {
+            id: '17',
+            description: 'desctiption',
+            createdAt: new Date('2021-03-15T23:00:00'),
+            link: 'link',
+            vendor: 'vendor',
+            hashTags: ['hashtag'],
+            discount: '10%',
+            validUntil: new Date('2021-06-30T23:00:00'),
+        },
+        
+        {
+            id: '18',
+            description: 'desctiption',
+            createdAt: new Date('2021-02-15T23:00:00'),
+            link: 'link',
+            vendor: 'vendor',
+            hashTags: ['hashtag'],
+            discount: '10%',
+            validUntil: new Date('2021-05-29T23:00:00'),
+        },
+        
+        {
+            id: '19',
+            description: 'desctiption',
+            createdAt: new Date('2021-01-15T23:00:00'),
+            link: 'link',
+            vendor: 'vendor',
+            hashTags: ['hashtag'],
+            discount: '10%',
+            validUntil: new Date('2021-04-29T23:00:00'),
+        },
+        
+        {
+            id: '20',
+            description: 'desctiption',
+            createdAt: new Date('2021-02-02T23:00:00'),
+            link: 'link',
+            vendor: 'vendor',
+            hashTags: ['hashtag'],
+            discount: '10%',
+            validUntil: new Date('2021-03-29T23:00:00'),
+        }
+
+    ];
+
+    function getAds(skip = 0, top = 10, filterConfig = undefined) {
+        if (typeof skip !== 'number' || typeof top !== 'number') {
+            console.log('Incorrect inputting type');
+            return;
+        }
+
+        if (filterConfig) {
+            let returningAds;
+            for (var param in filterConfig) {
+                if (param === 'hashTags') {
```

Comment:
What if `hashTags` isn't array? 🤔

Reply:
i guess it's always array... maybe with 1 element but still array